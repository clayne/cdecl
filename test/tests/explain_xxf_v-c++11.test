cdecl @ @ -xc++11 @ explain constexpr void f() @ 65
