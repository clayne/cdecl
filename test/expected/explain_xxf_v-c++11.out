explain constexpr void f()
                       ^
24: error: "constexpr void" is illegal until C++14
